### DISCUSSION TITLE
prefer-destructuring undestructurable ?

### DISCUSSION THREAD

**Discussion by Zenoo** [eslint/eslint#14669] [Q&A]
ESLint is asking me to destrucure an array, but I have no idea why or how.  
I assign an object stored in an array to a new variable, I don't see how it's destructurable ^^'  

![](https://i.imgur.com/z1ziklK.png)

**Comment by aladdin-add** [comment]
you can just use `eslint-disable`. :)

it sounds a reasonable enhancement to ignore these cases by default (or behind an option). can you file an issue, thanks!

**Reply by Zenoo** [reply]
I don't really like using `eslint-disable`, since it's kind of an _I know there's an issue but I won't fix it_ kinda deal, I would much rather fix the warning instead ^^'

I created the issue #14678 about this
